src/components/headings/H1.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/variables';
+
+h1.bdl-H1 {
+    @include font-family;
+
+    color: $bdl-gray;
+    font-size: 22px;
+    font-weight: bold;
+    letter-spacing: .3px;
```

Comment:
I am wondering if one should create a common mixin for
- the font family
- the bdl gray color
- the letter spacing

And override the one that has it different

Reply:
Each should include overrides for https://github.com/box/box-ui-elements/blob/master/src/styles/_reset.scss#L92
As well should include https://github.com/box/box-ui-elements/blob/master/src/styles/_variables.scss#L182 because any of the Hs can be portaled/tethered outside the parent container

Reply:
Also make sure that all the rules here like margin https://github.com/box/box-ui-elements/blob/master/src/styles/common/_typography.scss#L32 are overriden or reset in these new stylesheets. Because we want to make that typography.scss irrelevant. In other words if its included on the page it should have no effect on these new components.

Reply:
Re: better-black style: I have a separate PR for the better black since I consider that as a breaking change 🤷‍♂. 
Re: Overrides: I tested on the style sheet which include all the resets and it works. 
Re: commonmixin, was thinking of that too but the common styles are very mix.. so I didn't create the mixin. One thing I do think it might be helpful is creating the H1 ~ H6 mixin so others might want to mix that in with their style?

Reply:
Because of uppercasing, it looks like only H5 is the outliner. For example letter spacing is the same in all, which is the same as common-typography which you need to include here. Likewise missing margin will be common. bdl-gray color is also common to all other than H5 H6

So can be

```scss
@mixin bdl-H { 
    @include common-typography;
    color: bdl-gray;
    margin: 8px 0;
    text-rendering: optimizeLegibility;
}

.bdl-H1 {
    @include bdl-H;
    font-size: 22px;
    font-weight: bold;
    line-height: 28px;
}

.bdl-H5 {
    @include bdl-H;
    color: $bdl-gray-62;
    font-size: 11px;
    letter-spacing: .6px;
    line-height: 16px;
    text-transform: uppercase;
}
```